examples/tutorials/colabs/rigid_object_tutorial.ipynb

```diff
@@ -169,7 +169,7 @@
     "    # [basics]\n",
     "\n",
     "    # load some object templates from configuration files\n",
-    "    sphere_template_id = obj_templates_mgr.load_object_configs(\n",
+    "    sphere_template_id = obj_templates_mgr.load_configs(\n",
```

Comment:
Note : "load_object_configs" still works, but is being deprecated, and will be removed at some point in the future